drivers/SmartThings/zwave-garage-door-opener/profiles/eco-zw-gdo-profile.yml

```diff
@@ -0,0 +1,81 @@
+name: eco-zw-gdo-profile
+components:
+  - id: main
+    capabilities:
+    - id: doorControl
+      version: 1
+    - id: temperatureMeasurement
+      version: 1
+    - id: firmwareUpdate
+      version: 1
+    - id: refresh
+      version: 1
+    categories:
+    - name: GarageDoor
+  - id: sensor
+    capabilities:
+    - id: contactSensor
+      version: 1
+    - id: battery
+      version: 1
+    - id: tamperAlert
+      version: 1
+    - id: healthCheck
+      version: 1
```

Comment:
@cjswedes remind me if this is available to drivers to manage via a capability, or is it something else?

Reply:
I expect this capability to not do anything for the device. It will be deprecated once DTHs are no longer a thing. It might even cause nil index errors when used in a driver; im not sure though. Online/offline status for zwave devices is not controllable via the driver at all.

Reply:
I was under the impression that this capability would check for incoming messages from the device with this capability. Thank you for letting me know I can't control it from the driver side. Would it be advised to remove this capability completely as it will cause future issues?

Reply:
I am under the impression we will be deprecating it once the move from DTHs to drivers 
has been completed.